src/common/TokenChip/index.tsx

```diff
@@ -10,14 +10,22 @@ interface Props extends HTMLAttributes<HTMLDivElement> {
   symbol: string
   address: string | undefined
   link?: string
+  size?: number | undefined
 }
 
-const TokenChip: FC<Props> = ({ name, symbol, link, address, ...rest }) => {
+const TokenChip: FC<Props> = ({
+  name,
+  symbol,
+  link,
+  address,
+  size,
+  ...rest
+}) => {
   const TokenChipInnerContent = useMemo(
     () => (
       <S.Root {...rest}>
         {address ? (
-          <TokenIcon address={address} />
+          <TokenIcon address={address} size={size} />
         ) : (
           <S.TokenChipIcon symbol={symbol} />
```

Comment:
cover case when address is undefined and custom size prop passed.
S.TokenChipIcon should be same size as TokenIcon.

```suggestion
          <S.TokenChipIcon symbol={symbol} size={size} />
```